composeApp/build.gradle.kts

```diff
@@ -33,11 +38,13 @@ kotlin {
         commonMain {
             dependencies {
                 implementation(compose.runtime)
+                implementation(compose.ui)
```

Comment:
The new webMain ComposeViewport API is here, so we need compose.ui dependency to be explicit.